main.js

```diff
@@ -2,6 +2,7 @@
 // DOM  items
 const navbar = document.querySelector('#navbar');
 const burger = document.querySelector('#burger');
+const navLinks = document.querySelector('.navbar__links li a');
 
 let showMenu = false;
```

Comment:
- [ ] [OPTIONAL] Great job on your mobile menu, it works fine and looks great however please make sure your mobile menu disappears when we expand to the desktop version for keeping the sake of consistency and responsiveness

<img width="1052" alt="Screenshot 2023-05-25 at 10 20 25 AM" src="https://github.com/sergio-usma/First_capstone_project/assets/44798044/41b6c3bb-f968-4dfd-b0b5-210dd44d4e68">

Reply:
Yes, I added the navbar-active modifier to the media query, so it's not shown in desktop versions.